scipy/spatial/_polygon_surface_area.py

```diff
@@ -0,0 +1,79 @@
+"""
+
+Polygon Surface Area Code
+
+.. versionadded:: 1.0.0
+
+"""
+
+import numpy as np
```

Comment:
New python files should include future imports:
`from __future__ import division, print_function, absolute_import`